lib/media/drm_engine.js

```diff
@@ -1326,7 +1326,8 @@ shaka.media.DrmEngine = class {
       if (this.currentDrmInfo_.keySystem == 'com.microsoft.playready' &&
           keyId.byteLength == 16 &&
           !shaka.util.Platform.isTizen() &&
-          !shaka.util.Platform.isVideoFutur()) {
+          !shaka.util.Platform.isVideoFutur() &&
+          !shaka.util.Platform.isWeb0S()) {
```

Comment:
Is this supposed to have a zero in it instead of the letter O?  Now I'm concerned about whether or not your solution works, since it doesn't look like you could have tested it with this typo in it.

Reply:
@joeyparrish 
We were mistaken for the value of the userAgent we use.
We will update the patch.

Sorry.